index.html

```diff
@@ -58,6 +46,7 @@ <h2>Premium Materials</h2>
             </p>
           </div>
         </div>
```

Comment:
I would suggest you to choose the id to be flex instead of grid because your have made the style flex for this container.